RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -630,7 +630,7 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Wählen Sie einen Parameter und benutzen Sie die Schaltflächen oder doppelklicken Sie um ihn zu bewegen oder zu löschen</value>
   </data>
   <data name="Restore" xml:space="preserve">
-    <value>Wiederherstellen</value>
+    <value>Wieder-herstellen</value>
```

Comment:
If I understand this correctly (and I'm not sure I do), then the line break issue should resolve itself once this view is ported to XAML. Let's not force the line break.